.packit.yml

```
@@ -0,0 +1,21 @@
+specfile_path: packit/libmodulemd.spec
+upstream_package_name: libmodulemd
+downstream_package_name: libmodulemd
+
+synced_files:
+  - .packit.yml
+  - .make_packit_specfile.sh
+
+actions:
+    post-upstream-clone: .make_packit_specfile.sh
```

You can try this one:

```suggestion
    post-upstream-clone: ./.make_packit_specfile.sh
```